lib/Prefork.php

```diff
@@ -30,6 +30,10 @@ class Prefork
      * @var callable lamda function that is called when a child is reaped
      */
     public $on_child_reap = null;
+    /**
+     * @var callable lamda fuction that decide action
```

Comment:
`lamda` should be `lambda`.
If `action` means a function to determine the parent process should spawn child processes or not, this comment should be so.